Integrations/integration-opswat-metadefender.yml

```diff
@@ -39,42 +39,82 @@ configuration:
   defaultvalue: "66"
   type: 0
   required: true
+- display: It is cloud
+  name: cloud
+  defaultvalue: "false"
```

Comment:
Instances which are already defined won't get the default value, you should handle it in the code.

Reply:
I already handled it:
"if (params.cloud) "
It will go to else

Reply:
`params.cloud` will be `undefined` for existing instances, as they won't get the default value you set for it. You should handle that before.

Reply:
And if it will be undefined to where it will go? To else...

Reply:
Yes, but it's bad practice.